src/main/java/dev/espi/protectionstones/commands/ArgUnclaim.java

```diff
@@ -70,7 +99,39 @@ public boolean executeArgument(CommandSender s, String[] args, HashMap<String, S
             PSL.msg(p, PSL.RENT_CANNOT_BREAK_WHILE_RENTING.msg());
             return false;
         }
+        return unclaimBlock(r, p);
+    }
+    @Override
+    public List<String> tabComplete(CommandSender sender, String alias, String[] args) {
+        return null;
+    }
+    private int tryParseInt(String arg){
+        int i = 1;
+        try{
+            i = Integer.parseInt(arg);
+        }catch(NumberFormatException ignore){
+            //ignore
+        }
+        return i;
+    }
+    private void displayPSRegions(CommandSender s,List<PSRegion> regions,int page){
+        List<TextComponent> entries = new ArrayList<>();
+        for (PSRegion rs : regions) {
+            String msg;
+            if (rs.getName() == null) {
+                msg = ChatColor.GRAY + "> " + ChatColor.AQUA + rs.getId();
+            } else {
+                msg = ChatColor.GRAY + "> " + ChatColor.AQUA + rs.getName() + " (" + rs.getId() + ")";
+            }
+            TextComponent tc = new TextComponent(ChatColor.AQUA + " [-] " + msg);
+            tc.setHoverEvent(new HoverEvent(HoverEvent.Action.SHOW_TEXT, new ComponentBuilder("Click to delete "+rs.getId()).create()));
+            tc.setClickEvent(new ClickEvent(ClickEvent.Action.SUGGEST_COMMAND, "/ps unclaim " + rs.getId()));
```

Comment:
Instead of `/ps`, use `ProtectionStones.getInstance().getConfig().base_command` (I think?) since servers may have it set to something else